### PR TITLE
fix: Adjust default format strings

### PR DESCRIPTION
**Pull request by HansAuger**
#### Description
Adjust format strings for all modules where format is already lazily executed

#### Motivation and Context

Referring to #2111

#### How Has This Been Tested?

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: src/configs/erlang.rs [hunk 1]
```diff
@@ -13,7 +13,7 @@ pub struct ErlangConfig<'a> {
 impl<'a> RootModuleConfig<'a> for ErlangConfig<'a> {
     fn new() -> Self {
         ErlangConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/erlang.rs:16]: And here
```suggestion
            format: "via [$symbol($version )]($style)",
```

File: src/configs/go.rs [hunk 2]
```diff
@@ -13,7 +13,7 @@ pub struct GoConfig<'a> {
 impl<'a> RootModuleConfig<'a> for GoConfig<'a> {
     fn new() -> Self {
         GoConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/go.rs:16]:
```suggestion
            format: "via [$symbol($version )]($style)",
```

File: src/configs/rust.rs [hunk 3]
```diff
@@ -13,7 +13,7 @@ pub struct RustConfig<'a> {
 impl<'a> RootModuleConfig<'a> for RustConfig<'a> {
     fn new() -> Self {
         RustConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/rust.rs:16]:
```suggestion
            format: "via [$symbol($version )]($style)",
```

File: src/configs/ruby.rs [hunk 4]
```diff
@@ -13,7 +13,7 @@ pub struct RubyConfig<'a> {
 impl<'a> RootModuleConfig<'a> for RubyConfig<'a> {
     fn new() -> Self {
         RubyConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/ruby.rs:16]:
```suggestion
            format: "via [$symbol($version )]($style)",
```

File: src/configs/nim.rs [hunk 5]
```diff
@@ -13,7 +13,7 @@ pub struct NimConfig<'a> {
 impl<'a> RootModuleConfig<'a> for NimConfig<'a> {
     fn new() -> Self {
         NimConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/nim.rs:16]:
```suggestion
            format: "via [$symbol($version )]($style)",
```

File: src/configs/helm.rs [hunk 6]
```diff
@@ -13,7 +13,7 @@ pub struct HelmConfig<'a> {
 impl<'a> RootModuleConfig<'a> for HelmConfig<'a> {
     fn new() -> Self {
         HelmConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/helm.rs:16]:
```suggestion
            format: "via [$symbol($version )]($style)",
```

File: src/configs/julia.rs [hunk 7]
```diff
@@ -13,7 +13,7 @@ pub struct JuliaConfig<'a> {
 impl<'a> RootModuleConfig<'a> for JuliaConfig<'a> {
     fn new() -> Self {
         JuliaConfig {
-            format: "via [$symbol$version]($style) ",
+            format: "via [$symbol$version ]($style)",
```
Comment (andytom) [inline src/configs/julia.rs:16]:
```suggestion
            format: "via [$symbol($version )]($style)",
```

**Review by andytom — CHANGES_REQUESTED** [review]

File: docs/config/README.md [hunk 8]
```diff
@@ -2155,7 +2155,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                     |
 | ---------- | ---------------------------------- | ----------------------------------------------- |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                      |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                      |
```
Comment (HansAuger) [inline docs/config/README.md:2158]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                      |
```

File: docs/config/README.md [hunk 9]
```diff
@@ -2119,7 +2119,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                      |
 | ---------- | ---------------------------------- | ------------------------------------------------ |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                       |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                       |
```
Comment (HansAuger) [inline docs/config/README.md:2122]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                       |
```

File: docs/config/README.md [hunk 10]
```diff
@@ -1681,7 +1681,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                           |
 | ---------- | ---------------------------------- | ----------------------------------------------------- |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module                             |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module                             |
```
Comment (HansAuger) [inline docs/config/README.md:1684]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module                             |
```

File: docs/config/README.md [hunk 11]
```diff
@@ -1410,7 +1410,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                       |
 | ---------- | ---------------------------------- | ------------------------------------------------- |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                        |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                        |
```
Comment (HansAuger) [inline docs/config/README.md:1413]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                        |
```

File: docs/config/README.md [hunk 12]
```diff
@@ -1265,7 +1265,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                      |
 | ---------- | ---------------------------------- | ------------------------------------------------ |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                       |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                       |
```
Comment (HansAuger) [inline docs/config/README.md:1268]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                       |
```

File: docs/config/README.md [hunk 13]
```diff
@@ -1229,7 +1229,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                    |
 | ---------- | ---------------------------------- | ---------------------------------------------- |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                     |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                     |
```
Comment (HansAuger) [inline docs/config/README.md:1232]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                     |
```

File: docs/config/README.md [hunk 14]
```diff
@@ -926,7 +926,7 @@ The module will be shown if any of the following conditions are met:
 | ---------- | ---------------------------------- | -------------------------------------------------------- |
 | `symbol`   | `" "`                             | The symbol used before displaying the version of erlang. |
 | `style`    | `"bold red"`                       | The style for the module.                                |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                               |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                               |
```
Comment (HansAuger) [inline docs/config/README.md:929]:
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                               |
```

File: docs/config/README.md [hunk 15]
```diff
@@ -771,7 +771,7 @@ when there is a csproj file in the current directory.
 
 | Option      | Default                                   | Description                                              |
 | ----------- | ----------------------------------------- | -------------------------------------------------------- |
-| `format`    | `"[$symbol$version( 🎯 $tfm)]($style) "`  | The format for the module.                               |
+| `format`    | `"[$symbol$version (🎯 $tfm )]($style)"`  | The format for the module.                               |
```
Comment (HansAuger) [inline docs/config/README.md:774]:
```suggestion
| `format`    | `"[$symbol($version )(🎯 $tfm )]($style)"`  | The format for the module.                               |
```

**Comment by HansAuger** [comment]
Must have been drunk xD
Resolved!

**Review by andytom — APPROVED** [review]
LGTM

**Comment by andytom** [comment]
Thanks for sorting this out @HansAuger

**Pull request merged**